ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2384,6 +2404,46 @@ class PartCentredForceHolder {
     planetarium_camera_adjuster_.should_transfer_camera_coordinates = true;
   }
 
+  private void LoadDrawStyles(ConfigNode draw_styles) {
+    if (draw_styles == null) {
+      return;
+    }
+    ConfigNode history_parameters = draw_styles.GetAtMostOneNode("history");
+    if (history_parameters != null) {
```

Comment:
Remove these if statements, use `history_parameters?.GetDrawMeow` instead (technically this might mean twice the null checks but I’d rather reduce boilerplate than optimize null checks in something that runs in `OnAwake`.

Reply:
Wait no, `?.` worked with the out parameters, but not now that we return values.

Reply:
Upon reflection, I think I preferred the single function that loaded the entire style node.

Please merge
```C#
UnityEngine.Color GetDrawColour(this ConfigNode node)
```
and
```C#
GLLines.Style GetDrawStyle(this ConfigNode node)
```
into
```C#
void GetDrawStyle(this ConfigNode node, out UnityEngine.Color colour, out GLLines.Style style)
````
at which point the callsites can become
```C#
history_parameters?.GetDrawStyle(history_colour, history_style)
```
etc.